creator-node/src/services/stateMachineManager/stateReconciliation/stateReconciliationUtils.js

```diff
@@ -125,7 +127,7 @@ const _issueSyncRequestsUntilSynced = async (
     logger.warn(`Duplicate sync request: ${JSON.stringify(duplicateSyncReq)}`)
     return
   } else if (!_.isEmpty(syncReqToEnqueue)) {
-    await queue.add({
+    await queue.add('manual-sync', {
```

Comment:
wait this logic only runs on manual syncs? i see its hardcoded on L121 but am confused

Reply:
yeah cause `_issueSyncRequestsUntilSynced` is only called for write quorum